# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../moneypenny/include-stdlib.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./src/globals.cpp.orig	2006-02-04 10:40:46.000000000 +0100
+++ ./src/globals.cpp	2006-03-15 18:56:21.000000000 +0100
@@ -18,6 +18,7 @@
  *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
  ***************************************************************************/
 
+ #include <stdlib.h>
  #include "globals.h"
 
  #include <qdir.h>
--- ./src/guiprintermanualsetup.cpp.orig        2006-02-04 10:40:46.000000000 +0100
+++ ./src/guiprintermanualsetup.cpp     2006-03-15 19:11:58.000000000 +0100
@@ -22,6 +22,7 @@
 // App includes
 #include "guiprintermanualsetup.h"
 #include <printersetupdb.h>
+#include <stdlib.h>
 #include "globals.h"

 // Qt includes
